src/lib.rs

```diff
@@ -381,14 +383,17 @@ impl usb_device::bus::UsbBus for UsbBus {
             usb.udien
                 .modify(|_, w| w.wakeupe().set_bit().suspe().clear_bit());
             usb.usbcon.modify(|_, w| w.frzclk().set_bit());
-            //TODO disable PLL
+            let pll = self.pll.borrow(cs);
+            pll.pllcsr.modify(|_, w| w.plle().clear_bit());
         });
     }
 
     fn resume(&self) {
         interrupt::free(|cs| {
             let usb = self.usb.borrow(cs);
-            //TODO enable PLL
+            let pll = self.pll.borrow(cs);
+            pll.pllcsr
+                .modify(|_, w| w.pindiv().set_bit().plle().set_bit());
```

Comment:
I think we need to wait for PLL lock here before accessing the USB registers.

Like from the example:

https://github.com/agausmann/atmega-usbd/blob/30fcf0398e9d225f086980898c34fc000c7a0a8c/examples/arduino_keyboard.rs#L47-L51

Reply:
Ok, I'll add that in with the other changes.